### DISCUSSION TITLE
Proposal - Randomized delay

### DISCUSSION THREAD

**Discussion by Kein** [victornpb/undiscord#463] [3. Ideas] [answered]
Follow up to https://github.com/victornpb/undiscord/issues/149

There is already a way to define custom delay and push it further if API responds negatively. What would be also nice to have a randomized delay between minX and maxY that will serve some kind of imitation of a period similar to when a human user has to scroll up and find new message to delete.

**tl;dr** avoid uniform delays between deletions, they are easily detectable.

**Comment by gohamstergo** [comment, marked as answer]
https://github.com/victornpb/undiscord/discussions/342
